.github/ISSUE_TEMPLATE/config.yaml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
Are you sure you want to disable blank issues? There are some types of requests that do not match "Bug" or "Feature". E.g. someone could have questions about how a feature can be used (https://github.com/hetznercloud/hcloud-cloud-controller-manager/issues/517)

Reply:
Note that these templates will only be used by projects that don't have **any** Issue templates configured. If a single file is present, none of the defaults will be shown.